server/src/main/java/io/spine/server/entity/EventFieldFilter.java

```diff
@@ -42,8 +42,8 @@
  * An {@link EventFilter} which allows all the events but trims some of their message fields.
  *
  * <p>By default, the input event is returned unchanged. Specify a {@link FieldMask} for an event
- * type in order to trim certain fields off the event messages (the fields in the mask are
- * the retained, all the others are cleared).
+ * type to trim certain fields off the event messages (the fields in the mask are
```

Comment:
As long as we're improving the text, can we avoid braces in the new version, please?

Reply:
It's not the braces which is the problem. This text is hard to understand because the sentence is long.